### PR TITLE
docs: emphasize gem backtrace exclusion patterns

### PR DESCRIPTION
**Pull request by e2**
Give better example for filtering with multiple gems

### CODE REVIEW

File: features/configuration/backtrace_exclusion_patterns.feature [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+
+  If you want to filter out backtrace lines belonging to specific gems, you can
+  use `config.filter_gems_from_backtrace` like so:
+
+  ```
+  config.filter_gems_from_backtrace "ignored_gem", "another_ignored_gem",
+  ```
+
+  For more control over which lines to ignore, you can assign a list of
+  patterns the `backtrace_exclusion_patterns` option.
```
Comment (e2) [inline features/configuration/backtrace_exclusion_patterns.feature]: should be "patterns *for* the"

File: features/configuration/backtrace_exclusion_patterns.feature [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 
-  To reduce the noise when diagnosing failures, RSpec excludes matching lines
-  from backtraces. The default exclusion patterns are:
+  To reduce the noise when diagnosing failures, RSpec can exclude lines
+  belonging to certain gems or matching given patterns.
+
+  If you want to filter out backtrace lines belonging to specific gems, you can
+  use `config.filter_gems_from_backtrace` like so:
+
+  ```
```
Comment (myronmarston) [inline features/configuration/backtrace_exclusion_patterns.feature]: I think this should be ```ruby so it gets syntax highlighted.

File: features/configuration/backtrace_exclusion_patterns.feature [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+  config.filter_gems_from_backtrace "ignored_gem", "another_ignored_gem",
```
Comment (myronmarston) [inline features/configuration/backtrace_exclusion_patterns.feature]: Is it worth wrapping this in `RSpec.configure do |config|...end` to show the full context? As it exits, it's barely different from what's in the paragraph above.

File: features/configuration/backtrace_exclusion_patterns.feature (hunk 1 above)
Comment (myronmarston) [inline features/configuration/backtrace_exclusion_patterns.feature]: Actually, should be "patterns to the", I think.  Besides assigning to `backtrace_exclusion_patterns`, you can also append to it, and that's probably worth calling out here.

**Comment by myronmarston** [comment]
Nice improvement, @e2, I left a few comments.  Ignore the build failure for now; it's due to a new aruba release that broke us.

**Comment by e2** [comment]
I didn't wrap in `RSpec.configure do (...) end` because I added the patterns example code. Besides, the examples below use `RSpec.configure` blocks - which clear any confusion.

**Comment by myronmarston** [comment]
Thanks, @e2.  One other thing I forgot to mention before: on relish, hand-wrapped markdown doesn't render well because it puts line breaks everywhere you put one rather than treating it as a free-flowing paragraph.  I used to push to a relish staging environment (https://relishapp.com/rspec-staging) so that I could see how it renders and adjust my line breaks accordingly until I realized it was simpler just to not put _any_ line breaks in the middle of a paragraph, letting relish's markdown render put them at the appropriate places.  So now I find it easier/better to write relish narrative paragraphs as one long unbroken line each.  Do you mind changing your paragraphs to that?

With that in place, this'll be good to merge.  Thanks!

**Comment by myronmarston** [comment]
Regarding the failed build...I just merged #1848 which fixes it, so do you mind rebasing against master so that the builds here include that?  Then we should be able to see a green build here which is always nice.

**Comment by e2** [comment]
I'm not sure if I understood correctly about the newlines - let me know if I didn't (I'm not sure). I tried not to change the original at first, so I don't see how I "introduced" any problems.

**Comment by myronmarston** [comment]
> I'm not sure if I understood correctly about the newlines - let me know if I didn't (I'm not sure)

That's not what I meant.  Maybe an example will make what I meant clearer (sorry that my description wasn't clear!).  Check out the long unwrapped lines that were added in this PR, particularly the linked line and discussion:

https://github.com/rspec/rspec-core/pull/1842/files#diff-55a2a0f3cabfec543152971929f30a32R15

https://github.com/rspec/rspec-core/pull/1842/files

**Comment by e2** [comment]
(facepalm) - I get it now.

Fixed.

Let me know if there's anything else.

**Comment by myronmarston** [comment]
:+1: Will merge when green.  Thanks!

**Comment by e2** [comment]
Thanks as well! I'll start with a PR next time.

**Pull request merged**